scripts/e2e-ci.sh

```diff
@@ -32,6 +32,14 @@ MERLINITE_GGUF_MODEL="merlinite-7b-lab-Q4_K_M.gguf"
 MISTRAL_GGUF_REPO="TheBloke/Mistral-7B-Instruct-v0.2-GGUF"
 MISTRAL_GGUF_MODEL="mistral-7b-instruct-v0.2.Q4_K_M.gguf"
 
+HUGGINGFACE_DIR="huggingface.co"
+
+MERLINITE_GGUF_REPO="instructlab/merlinite-7b-lab-GGUF"
+MERLINITE_GGUF_MODEL="merlinite-7b-lab-Q4_K_M.gguf"
+
+CACHE_HOME=$(python -c 'import platformdirs; print(platformdirs.user_cache_dir())')
+MODEL_CACHE="${CACHE_HOME}/instructlab/models"
```

Comment:
Why is this not a part of `init_e2e_tests`?

Reply:
updated